python/ray/tests/test_plasma_unlimited.py

```diff
@@ -247,6 +248,34 @@ def test_fallback_allocation_failure(shutdown_only):
 #     finally:
 #         ray.shutdown()
 
+
+@pytest.mark.skipif(
+    platform.system() == "Windows", reason="Failing on Windows.")
+def test_plasma_allocate(shutdown_only):
+    ray.init(
+        object_store_memory=300 * 1024**2,
+        _system_config={
+            "max_io_workers": 4,
+            "automatic_object_spilling_enabled": True,
+        },
+        _temp_dir="/tmp/ray/test_tmp_dir")
+    res = []
+    data = np.random.randint(
+        low=0, high=256, size=(90 * 1024**2, ), dtype=np.uint8)
+    for _ in range(3):
+        res.append(ray.put(data))
+    # keep reference for second and third object, force evict first object
+    _ = ray.get(res[1:])  # noqa
+    # keep reference for fourth object, avoid released by plasma GC.
+    __ = ray.put(data)  # noqa
+
+    # Check fourth object whether allocate to disk
```

Comment:
Can you use `check_spilled_mb` to see if it was disk allocated instead?

Reply:
done.